iam-login-service/src/test/java/it/infn/mw/iam/test/api/account/find/FindAccountIntegrationTests.java

```diff
@@ -290,4 +291,14 @@ public void findNotInGroupWorks() throws Exception {
       .andExpect(jsonPath("$.Resources[1].id", is("bffc67b7-47fe-410c-a6a0-cf00173a8fbb")));
   }
 
+  @Test
+  public void findByCertificateSubjectAndIssuerWorks() throws Exception {
+
+    mvc.perform(get(FIND_BY_CERT_SUBJECT_AND_ISSUER_RESOURCE).param("certificateSubject", "CN=test2,O=IGI,C=IT").param("certificateIssuer", "CN=Test CA,O=IGI,C=IT"))
+      .andExpect(OK)
+      .andExpect(jsonPath("$.totalResults", is(1)))
+      .andExpect(jsonPath("$.Resources[0].userName", is("admin")));
+
+  }
+
```

Comment:
I'd add a "notFound" test with fake issuer and right subject, a fake subject and right issuer and both fake items.